File: _IFSP/LP1/Lista_2/Exercicios_SALA-lista-2/EX4-Triangulo-s/EX4-s.py
l1 = float(input())
l2 = float(input())
l3 = float(input())
st = "NÃO É UM"
tp = ""
if l1 < l2 + l3 and l2 < l1 + l3 and l3 < l1 + l2:
    st = "É UM"
    if l1 == l2 == l3:
        tp = "EQUILÁTERO"
    elif l1 != l2 and l2 != l3 and l1 != l3:
        tp = "ESCALENO"
    else:
        tp = "ISÓSCELES"
print(f"{st} triângulo {tp}")

File: 01_IFSP/LP1/Lista_2/Exercicios_SALA-lista-2/EX4-Triangulo-s/EX4-s.c
#include <stdio.h>

int main() {
    float l1, l2, l3;

    scanf("%f", &l1);
    scanf("%f", &l2);
    scanf("%f", &l3);

    if (l1 < l2 + l3 && l2 < l1 + l3 && l3 < l1 + l2) {
        if (l1 != l2 && l2 != l3 && l1 != l3) {
            printf("É um triângulo Escaleno\n");
        } else if (l1 == l2 && l2 == l3) {
            printf("É um triângulo Equilátero\n");
        } else {
            printf("É um triângulo Isósceles\n");
        }
    } else {
        printf("Não é um triângulo\n");
    }

    return 0;
}
